client/Routing/Models.elm

```diff
@@ -98,6 +98,10 @@ routingConfig =
     ]
 
 
+
+-- TODO use https://package.elm-lang.org/packages/elm/url/1.0.0/Url-Builder
```

Comment:
todo?

Reply:
This TODO I would keep for now if you may. This will be a too big change for already gigantic PR.